app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><%#= this meta tag is per Bootstrap: https://getbootstrap.com/docs/4.4/getting-started/introduction/  %>
+    // NB: please refer to the section in README.md ## A note on js packs and stylesheets
```

Comment:
Looks like a JS comment ended up on a ruby file. Might have something to do with the failing system test, its failed all 3 times I ran it

Reply:
Usually I try to put ruby comments in my JavaScript...

ok fixed now